rel/mim1.vars-toml.config

```diff
@@ -32,6 +32,11 @@
 [[host_config]]
   host_type = \"test type\"
 
+  [host_config.access]
```

Comment:
Note: if we merge https://github.com/esl/MongooseIM/pull/3261 first, no changes would be needed in this file.

Reply:
Apply all other changes except this one, let's merge that other PR first then.